integration-tests/tests/advanced-happy-path.spec.ts

```diff
@@ -0,0 +1,168 @@
+import { applicationDetailPagePO, ComponentsPagePO } from '../support/pageObjects/createApplication-po';
+import { ApplicationDetailPage } from '../support/pages/ApplicationDetailPage';
+import { ComponentPage } from '../support/pages/ComponentsPage';
+import { LatestCommitsTabPage } from '../support/pages/tabs/LatestCommitsTabPage';
+import { PipelinerunsTabPage } from '../support/pages/tabs/PipelinerunsTabPage';
+import { Applications } from '../utils/Applications';
+import { Common } from '../utils/Common';
+
+describe.skip('Create Components using the UI', () => {
```

Comment:
Please comment the entire spec file as the skipping test is causing issue on uploading the result on Report Portal which leads to launch timeout.

Reply:
Yes, just did that, thanks!